src/main/java/frc/robot/RobotContainer.java

```diff
@@ -23,8 +23,8 @@
 public class RobotContainer {
   // Subsystems
   private final Drivetrain drivetrain = Drivetrain.getInstance();
-  // private final Arm arm = new Arm();
-  // private final Claw claw = new Claw();
+  private final Arm arm = Arm.getInstance();
+  private final Claw claw = Claw.getInstance();
```

Comment:
Could you keep this commented out for now? Just because the robot doesn't have these subsystems built yet.